### PR TITLE
Suggestion: New Event after eMail was sent

### PR DESCRIPTION
**Pull request by stanschus**
Hi everyone,
I'm slowly becoming the "add-new-events"-guy ;)

reason for adding this event: I'm attaching (form generated) files to an email. after email with given attachment was sent, generated file(s) should be deleted. I'm trying to avoid storing data on the server as much as I can. maybe this will come in handy for others too in the future.

### CODE REVIEW

File: email.php [hunk 1]
```diff
@@ -165,6 +165,9 @@ protected function sendFormEmail($form, $params, $vars)
 
         // Send e-mail
         $this->email->send($message);
+
+        //fire event after eMail was sent
+        $this->grav->fireEvent('onEmailWasSent', new Event(['message' => $message, 'params' => $params, 'form' => $form]));
```
Comment (rhukster) [inline email.php:170]: For consistency, i think we should change this to `onEmailSent` . i know its close to `onEmailSend` but we don't use `was` anywhere in our event names.

**Review by rhukster — CHANGES_REQUESTED** [review]

File: email.php (hunk 1 above)
Comment (stanschus) [reply email.php]: Done.
Hopefully I'm not coming up with any other events in the future, I'm getting the feeling we are running out of event names ;)

**Comment by stanschus** [comment]
@rhukster just wanna ping you, ask if changes are okay. thanks :)

**Comment by rhukster** [comment]
Sorry missed that you updated this.  Looks good to me now, and merged for next release.  Cheers.

**Pull request merged**